utilities/include/alps/numeric/is_zero.hpp

```diff
@@ -10,6 +10,8 @@
 #define ALPS_NUMERIC_IS_ZERO_HPP
 
 #include <cmath>
+#include <complex>
```

Comment:
Do we need `#include <complex>` here?
```suggestion
```

Reply:
I think we need <complex> here, since we have a specialized version of is_zero_helper for std::complex<T>.